src/index.js

```diff
@@ -150,14 +150,17 @@ const processor = postcss.plugin('postcss-modules-scope', function(options) {
     let exportedNames = Object.keys(exports);
     if (exportedNames.length > 0) {
       css.append(postcss.rule({
-        selector: `:export`,
-        nodes: exportedNames.map(exportedName => postcss.decl({
+        selector: `:export`
+      }));
+      let exportRule = css.nodes[css.nodes.length - 1];
```

Comment:
Why not just:

```
const exportRule = postcss.rule({selector: `:export`});
exportedNames.forEach(name => exportRule.append(/*some code*/));
css.append(exportRule);
```

that `css.nodes[css.nodes.length - 1]` reference looks like private-ish api that should not be used